### ISSUE TITLE
add support for unrealircd bans

### ISSUE THREAD

**Issue by thommey** [eggheads/eggdrop#49]
| Trac | Data |
| --: | :-- |
| Ticket | 49 |
| Reported by | @thommey |
| Status | closed |
| Component | Core |
| Priority | minor |
| Version | 1.6.20 |

"+b ~c:#chan" _sigh_
depends on #3

**Comment by thommey** [comment]
**Comment by @thommey**
duplicate of #18
